examples/esp32-wroom-32/slipdev/slipdev_params.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
```

Comment:
i think this should be changed , i don't know if it fit in our documentation build system